security/firely-auth/firely-auth-settings.rst

```diff
@@ -94,27 +94,14 @@ To make Firely Server known to Firely Auth, fill in the ``FhirServerConfig``:
     E.g. in Postman you can provide this parameter by adding it to the Auth URL, like this: ``{{ids}}/connect/authorize?aud=http://localhost:4080`` 
     See the ``aud`` parameter in `SMART on FHIR authorization request`_
 
+- ``IntrospectionSecret``: When using a :term:`reference token`, Firely Server must verify the token with Firely Auth and the communication needs to be authenticated by providing the name and the secret. This configuration is only needed if at least one :term:`client` is configured to use reference tokens, see :ref:`firely_auth_settings_tokentypes` for the configuration.
```

Comment:
It seems that we have to provide a value for that, independently of the client token types